File: backend/Models/Reader.cs
using System.ComponentModel.DataAnnotations;

namespace lab8.Models
{
    public class Reader
    {
        [Key]
        public int ID { get; set; }
        [Required]
        public string Surname { get; set; } = null!;
        [Required]
        public string Name { get; set; } = null!;
        [Required]
        public string PatronymicName { get; set; } = null!;
        [Required]
        public string Address { get; set; } = null!;

        public int Phone { get; set; }
        public DateTime DateBirth { get; set; }
    }
}

File: frontend/app/models/reader.ts
export class Reader{
    ID!: number;
    Surname!: string;
    Name!: string;
    PatronymicName!: string;
    Address!: string;
    Phone!: number;
    DateBirth!: Date;

    constructor(id: number, surname: string, name: string, patronymicName: string, address: string, phone: number, dateBirth: Date){
        this.ID = id;
        this.Surname = surname;
        this.Name = name;
        this.PatronymicName = patronymicName;
        this.Address = address;
        this.Phone = phone;
        this.DateBirth = dateBirth;

    }
}